artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/LinkedListImpl.java

```diff
@@ -29,6 +32,9 @@
  */
 public class LinkedListImpl<E> implements LinkedList<E> {
 
+   // TODO: use an object placement for fast delete messages with an id from the linkes list
+   Map<Object, Node<E>> objectPlacement;
```

Comment:
Another map? :S -- whats the number of Objects here expected? I thought this was the purpose of nodeMap?

Reply:
to that i can't see it being used...

Reply:
Yeah.. I'm removing on my next review commit from Robbie Gemmel